test.js

```diff
@@ -1,80 +1,71 @@
-'use strict';
-var assert = require('assert');
-var fs = require('fs');
-var globby = require('./');
+import fs from 'fs';
+import test from 'ava';
+import m from './';
```

Comment:
I'm 👎  on renaming `globby` -> `m`. Users often look for usage examples in the tests file; it should be optimized for readability.
Any strong reasons to prefer this other naming pattern?